.travis.yml

```diff
@@ -1,19 +1,26 @@
 language: go
 
+branches:
+  except:
+    - master
+  only:
+    - release
 go:
   - 1.13.x
 
 services:
   - docker
 
 script:
-  - make
+  - go test ./...
+  - curl -sfL https://git.io/goreleaser | sh -s -- check
```

Comment:
curl -sfL https://git.io/goreleaser is use in the below as well, do you wan to define a variable for this and used everywhere?

Reply:
@mallow111 actually i should remove this line, it was just there for a validation. We dont need this line